--- alt-aspell-pl-20040916/configure.orig	2004-09-17 02:47:30.263333952 +0200
+++ alt-aspell-pl-20040916/configure	2004-09-17 02:48:12.548905568 +0200
@@ -68,11 +68,11 @@
   then WORD_LIST_COMPRESS=word-list-compress; fi
 
 echo $ECHO_N "Finding Dictionary file location ... $ECHO_C"
-dictdir=`$ASPELL dump config | grep "# dict-dir current" | cut -d : -f 2 | tr -d " "`
+dictdir=`$ASPELL dump config dict-dir`
 echo $dictdir
 
 echo $ECHO_N "Finding Data file location ... $ECHO_C"
-datadir=`$ASPELL dump config | grep "# data-dir current" | cut -d : -f 2 | tr -d " "`
+datadir=`$ASPELL dump config data-dir`
 echo $datadir
 
 echo "ASPELL = $ASPELL" > Makefile
